### DISCUSSION TITLE
Razer Tartarus v2 works

### DISCUSSION THREAD

**Discussion by KayTaneJupiter** [sezanzeb/input-remapper#164] [Show and tell]
Just documenting for future people who search for it.

All buttons of the Razer Tartarus v2 can be mapped with key-mapper 1.0

OpenSUSE Tumbleweed.

**Comment by hardbrocklife** [comment]
Bumping to show love. IR allowed me to transition to manjaro from windows when one of the few things holding me back was Razer Tartarus v2 button mapping. Being able to moving my gaming environment to manjaro 1:1 was the only thing holding me back.

**Reply by Mcdonnellpatrick** [reply]
How did you guys get this to work? Would you be able to post a screenshot of your layout please?

**Reply by KayTaneJupiter** [reply]
> How did you guys get this to work? Would you be able to post a screenshot of your layout please?

Actually, a recent version broke the working-ness. At least, the last time I tried it.

**Comment by hardbrocklife** [comment]
Sure thing. Its being weird right now and doesnt want to open. When it does Ill show you some of what Ive made.

**Comment by hardbrocklife** [comment]
I think an application ate a dependency so I had to rebuild it.

![image](https://github.com/sezanzeb/input-remapper/assets/53183711/8b4de4e8-77a1-46a3-914f-c5fe1bf12e16)

This is what mine looks like. Obvs the input on the left and output on the right. To add a maping click Add > Record > then enter what you want on the right side. It has like its own language, which it shows you when you click a button on something and look at the bottom of the output box. 

I mapped alt-tab to my 1 key:

![image](https://github.com/sezanzeb/input-remapper/assets/53183711/2045ca46-ebd3-4838-9828-06eeab2193b3)

and esc to my 3 key:

![image](https://github.com/sezanzeb/input-remapper/assets/53183711/35374e9b-7c8a-428b-a64e-75ec1c4b3bbd)


If you want to do more complicated macros it has like its own language. Like if I wanted the macro so spell Hard, I would enter a macro that looks like this "modify(Shift_L, key(h)).key(a).key(r).key(d)". A "," indicates the key press will modify the entry after the "," and "." separates key presses. 

If you click Add > Record > 2 it will enter the 2 as a new mapping and allow for editing the output on the right.
